cli/tests/pcluster/api/controllers/test_cluster_compute_fleet_controller.py

```diff
@@ -76,6 +79,27 @@ def test_successful_request(self, mocker, client, scheduler, status, last_status
             mocker.patch("pcluster.aws.dynamo.DynamoResource.get_item", return_value=dynamodb_item)
             # mock the method to update the item in dynamodb
             mocker.patch("pcluster.aws.dynamo.DynamoResource.put_item")
+        elif scheduler == "plugin":
```

Comment:
this logic seems repeated in multiple places. Would it make sense to put it in common?

Reply:
tnx, refactored